site/source/docs/api_reference/module.rst

```diff
@@ -103,3 +103,21 @@ Other methods
 
 	:param obj: The JavaScript-wrapped C++ object to be destroyed.
 	
+Overriding execution environment
+================================
+
+``Module`` is able to detect its execution environment by checking the presence of some typical objects of the environment itself (such as ``window`` for browser)
```

Comment:
It's not "Module" that is able to detect, but rather the entire emscripten-generated program.